### PR TITLE
Show the type of save that was performed for the record instead of just showing either Saved/Published.

### PR DESCRIPTION
**Pull request by bergice**
See https://github.com/silverstripe/silverstripe-cms/issues/2448

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "node": "^10.x"
   },
   "scripts": {
-    "build": "yarn && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && NODE_ENV=production webpack -p --bail --progress",
```
Comment (robbieaverill) [inline package.json:12]: Can you revert this change? It doesn't seem relevant to this PR

**Comment by Cheddam** [comment]
It should be safe to bump the requirements to `4.7@dev`, regardless of whether the branches exist yet - Composer will resolve it to `4`.

**Comment by robbieaverill** [comment]
I don't know if that's true, given that we've removed all the branch aliases

**Comment by Cheddam** [comment]
Looks like the bump worked - @robbieaverill are you happy to merge?
